static/app/actionCreators/prompts.tsx

```diff
@@ -144,10 +144,12 @@ export function usePrompts({
   projectId,
   daysToSnooze,
   options,
+  isDismissedOverride = null, // if not provided, use the default promptIsDismissed function
```

Comment:
```suggestion
  isDismissed = promptIsDismissed,
```
And then later on, call `isDismissed` everywhere, that way you don't need to handle duplicate dismiss functionalities